ruleset/sca/rhel/9/cis_rhel9_linux.yml

```diff
@@ -453,236 +527,259 @@ checks:
     compliance:
-      - cis: ["1.2.3"]
-      - cis_csc: ["3.4", "7.3"]
-      - pci_dss: ["6.2"]
-      - nist_800_53: ["SI.2", "SA.11", "SI.4"]
-      - gpg_13: ["4.3"]
-      - gdpr_IV: ["35.7.d"]
-      - hipaa: ["164.312.b"]
-      - tsc: ["A1.2", "CC6.8"]
+      - cis: ["1.2.2"]
+      - cis_csc_v8: ["7.3"]
+      - cis_csc_v7: ["3.4"]
+      - nist_sp_800-53: ["SI-2"]
+      - iso_27001-2013: ["A.9.1.1"]
+      - cmmc_v2.0: ["SI.L1-3.14.1"]
+      - pci_dss_3.2.1: ["6.2"]
+      - soc_2: ["CC7.1"]
+      - mitre_techniques: ["T1195","T1195.001"]
+      - mitre_tactics: ["TA0005"]
     condition: all
     rules:
       - "f:/etc/dnf/dnf.conf -> r:gpgcheck=1"
       - 'not c:grep -Rh ^gpgcheck /etc/yum.repos.d/ -> r:gpgcheck\s*=\s*0'
 
-  # 1.2.4 Ensure package manager repositories are configured (Manual) - Not implemented
+  # 1.2.3 Ensure package manager repositories are configured (Manual) - Not implemented
+  # 1.2.4 Ensure repo_gpgcheck is globally activated (Manual) - Not implemented
+
 
   ###############################################
   # 1.3 Filesystem Integrity Checking
   ###############################################
 
   # 1.3.1 Ensure AIDE is installed (Automated)
-  - id: 28034
+  - id: 28027 
     title: "Ensure AIDE is installed."
     description: "Advanced Intrusion Detection Environment (AIDE) is a intrusion detection tool that uses predefined rules to check the integrity of files and directories in the Linux operating system. AIDE has its own database to check the integrity of files and directories. AIDE takes a snapshot of files and directories including modification times, permissions, and file hashes which can then be used to compare against the current state of the filesystem to detect modifications to the system."
     rationale: "By monitoring the filesystem state compromised files can be detected to prevent or limit the exposure of accidental or malicious misconfigurations or modified binaries."
     remediation: "Run the following command to install AIDE: # dnf install aide Configure AIDE as appropriate for your environment. Consult the AIDE documentation for options. Initialize AIDE: Run the following commands # aide --init # mv /var/lib/aide/aide.db.new.gz /var/lib/aide/aide.db.gz."
     compliance:
       - cis: ["1.3.1"]
-      - cis_csc: ["3.14", "14.9"]
-      - pci_dss: ["2.2.4"]
-      - nist_800_53: ["CM.1"]
-      - tsc: ["CC5.2"]
+      - cis_csc_v8: ["3.14"]
+      - cis_csc_v7: ["14.9"]
+      - nist_sp_800-53: ["AU-2"]
+      - cmmc_v2.0: ["AC.L2-3.1.7"]
+      - hipaa: ["164.312(b)","164.312(c)(1)","164.312(c)(2)"]
+      - pci_dss_3.2.1: ["10.2.1","11.5"]
+      - pci_dss_4.0: ["10.2.1","10.2.1.1"]
+      - soc_2: ["CC6.1"]
+      - mitre_techniques: ["T1565","T1565.001"]
+      - mitre_tactics: ["TA0001"]
     condition: all
     references: "AIDE stable manual: http://aide.sourceforge.net/stable/manual.html"
```

Comment:
The reference field should be an array:

Current:

```
  references: "AIDE stable manual: http://aide.sourceforge.net/stable/manual.html"
```


Expected:

```
  references: 
    - "AIDE stable manual: http://aide.sourceforge.net/stable/manual.html"
 ```

Reply:
Done.